editor/plugin.js

```diff
@@ -308,8 +308,11 @@ org.ekstep.questionset.EditorPlugin = org.ekstep.contenteditor.basePlugin.extend
   },
   getSummary: function() {
     var instance = this;
-    var summary = {'totalQuestions': 0,'totalScore': 0};
+    var summary = {'totalQuestions': 0,'totalScore': 0, 'questions': []};
     var totalQuestionsToRender = instance.config.total_items;
+    instance._questions.forEach(function(question,key) {
+      summary.questions.push({'identifier': question.identifier});
```

Comment:
While editing the content(Question set), will we having this question.identifier field? Is it verified?